### PR TITLE
Add createdAt to OAS response body

### PR DESCRIPTION
**Pull request by christineyost**
This PR is for [API-14250](https://vajira.max.gov/browse/API-14250). The createdAt field was added to the OASResponseBody so that it will be included in the response from the `GET /docserver/apis/<API_NAME>/versions/<API_VERSION>/oas` endpoint.

### CODE REVIEW

**Review by mheusgit — APPROVED** [review]
Looks good

**Review by dsteeve — DISMISSED** [review]

File: test/oas/oas.controller.test.ts [hunk 1]
```diff
@@ -206,6 +206,7 @@ describe('OASController', () => {
         transformedSpec: 'Breakfast 2',
         sfTransformedSpec: 'Breakfast 2',
         iteration: 2,
+        createdAt: '2022-01-01T21:43:52.000Z',
```
[the remark this answers is not in the document]
Comment (christineyost) [reply test/oas/oas.controller.test.ts:211]: Updated this so that the oas variable also has createdAt

**Review by mheusgit — APPROVED** [review]
Still good

**Review by dsteeve — APPROVED** [review]
looks great!

**Pull request merged**